legal-api/report-templates/template-parts/business-summary/foreignJurisdiction.html

```diff
@@ -37,9 +37,8 @@
         </tr>
     </table>
 </div>
-
-{% if continuationIn.mode == 'EXPRO' %}
-    {% if continuationIn.status == 'HISTORICAL' %}
+{% if continuationIn.expro_identifier == 'EXPRO' %}
```

Comment:
expro_identifier will be a business identifier in the colin (`A12324567`). You can check if it has a value

Reply:
unsure of what to change this to

Reply:
```suggestion
{% if continuationIn.expro_identifier %}
```